### PR TITLE
Add note for installing Minikube vs Minishift vs Openshift

### PR DESCRIPTION
**Pull request by aliok**
Rebasing https://github.com/redhat-developer-demos/knative-tutorial/pull/143

Also fixed a minor copy to clipboard problem

### CODE REVIEW

File: documentation/modules/ROOT/pages/01-setup.adoc [hunk 1]
```diff
@@ -6,6 +6,8 @@ include::_attributes.adoc[]
 
 The following CLI tools are required for running the exercises in this tutorial. Please have them installed and configured before you get started with any of the tutorial chapters.
 
+You don't have to have both `Minikube` and `Minishift`. Also, if you have an existing OpenShift cluster you can also use it.
```
Comment (kameshsampath) [inline documentation/modules/ROOT/pages/01-setup.adoc:9]: @aliok - wondering if still need this note, as the current documentation has split vanilla kubernetes and openshift instructions.

File: documentation/modules/ROOT/pages/_partials/knative-objects.adoc [hunk 2]
```diff
@@ -98,19 +98,19 @@ kubectl --namespace {tutorial-namespace} get rev \
 --label=serving.knative.dev/service=greeter \
 --sort-by="{.metadata.creationTimestamp}"
 ----
-copyToClipboard::knative-revisions[]
+copyToClipboard::{doc-sec}knative-revisions[]
```
Comment (kameshsampath) [inline documentation/modules/ROOT/pages/_partials/knative-objects.adoc]: i would like modify it for standard pattern i follow like `{doc-sec}-<clipboard id>`

File: documentation/modules/ROOT/pages/_partials/knative-objects.adoc [hunk 3]
```diff
@@ -98,19 +98,19 @@ kubectl --namespace {tutorial-namespace} get rev \
...
 --
 oc::
 +
 --
-[#{doc-sec}-oc-knative-revisions]
+[#{doc-sec}oc-knative-revisions]
```
Comment (kameshsampath) [inline documentation/modules/ROOT/pages/_partials/knative-objects.adoc]: same as above

File: documentation/modules/ROOT/pages/_partials/knative-objects.adoc [hunk 4]
```diff
@@ -98,19 +98,19 @@ kubectl --namespace {tutorial-namespace} get rev \
...
 [source,bash,subs="+macros,+attributes"]
 ----
 oc --namespace {tutorial-namespace} get rev \
  --label=serving.knative.dev/service=greeter \
  --sort-by="{.metadata.creationTimestamp}"
 ----
-copyToClipboard::oc-knative-revisions[]
+copyToClipboard::{doc-sec}oc-knative-revisions[]
```
Comment (kameshsampath) [inline documentation/modules/ROOT/pages/_partials/knative-objects.adoc]: same as above

**Review by kameshsampath — COMMENTED** [review]
@aliok - thanks for catching these issues.

File: documentation/modules/ROOT/pages/01-setup.adoc (hunk 1 above)
Comment (aliok) [reply documentation/modules/ROOT/pages/01-setup.adoc:9]: ok, sure. but, still saying you need minishift is kind of misleading if a person is only interested in minikube path.

When I did the tutorial for the first time, I was thinking "oh, ok I need both minikube and minishift. probably later sections have something that's only working on OpenShift".

not a strong opinion, not a big thing. ok to drop this.

File: documentation/modules/ROOT/pages/_partials/knative-objects.adoc (hunk 2 above)
Comment (aliok) [reply documentation/modules/ROOT/pages/_partials/knative-objects.adoc]: ok, gonna update my PR. the problem was that the copy to clipboard functionality was not working. pattern doesn't matter.

Comment (aliok) [reply documentation/modules/ROOT/pages/_partials/knative-objects.adoc]: updated now

**Comment by aliok** [comment]
@kameshsampath fixed the `doc-sec` pattern with a new commit now.

Let me know if you want to keep or delete the note about minishift vs minikube and I can send a new commit deleting that note.

**Comment by kameshsampath** [comment]
Will add the note also buddy. Am doing some refactor of thr chapters as @burrsutter wanted to move some chapters to advanced section. Will cherry pick these into them. Thanks for the PR.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by kameshsampath** [comment]
stale and no longer applicable. fixed it already. thanks @aliok

**Pull request closed without merging**